File: web3rpc/rpc-specs/code-samples/python/debug/others/getModifiedStorageNodesByNumber.py
from web3 import Web3
from web3py_ext import extend

host = "https://dev.api.klaytn.sotatek.works"

address = "0x73a7d19d14f7dfac5b799e405e22133b2adc57a6"
startBlockNum = 100
endBlockNum = 200

w3 = Web3(Web3.HTTPProvider(host))
debug_response = w3.debug.get_modified_storage_nodes_by_number(address, startBlockNum, endBlockNum)

print(debug_response)

File: web3rpc/rpc-specs/code-samples/javascript/debug/others/getModifiedStorageNodesByNumber.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  const address = "0x73a7d19d14f7dfac5b799e405e22133b2adc57a6";
  const startBlockNum = 100;
  const endBlockNum = 200;

  provider.debug
    .getModifiedStorageNodesByNumber(
      address,
      startBlockNum,
      endBlockNum,
      {},
      (err, data, response) => {},
    )
    .then((data) => {
      console.log(data);
    });
})();
